src/main/resources/logback.xml

```diff
@@ -0,0 +1,6 @@
+<configuration scan="true">
+	<property scope="local" name="app.context.name" value="student-report-processor" />
+	<property scope="local" name="app.base.package.name" value="org.opentestsystem.delivery.studentreportprocessor" />
+	<property scope="local" name="mna.appender.active" value="true" />
+	<include resource="logback-included-common-config.xml" />

```

Comment:
The centralized logging configuration has been located in a single place so far.

https://github.com/SmarterApp/SS_SharedCode/blob/develop/src/main/resources/logback-included-common-config.xml